domain/credentials/service.go

```diff
@@ -64,11 +64,13 @@ func (s Service) ManageNutsOrgCredential(customer domain.Customer, shouldHaveCre
 }
 
 func (s Service) GetCredentials(customer domain.Customer) ([]domain.OrganizationConceptCredential, error) {
-	// not enough info, return
-	if customer.City == nil {
-		return []domain.OrganizationConceptCredential{}, nil
+	searchBody := vcrApi.SearchJSONRequestBody{
+		Params: []vcrApi.KeyValuePair{
+			{Key: "subject", Value: customer.Did},
```

Comment:
aaaah now I understand. Interesting way to find the customer's credentials.. :-p